Src/F_Interfaces/Base/AMReX_distromap_fi.cpp

```diff
@@ -28,6 +28,14 @@ extern "C" {
 	dmo = new DistributionMapping(*dmi);
     }
 
+    void amrex_fi_distromap_get_pmap (DistributionMapping*& dm, int* pmap, const int plen)
```

Comment:
Could we change `DistributionMapping*&` to `const DistributionMapping*` and make another change in Fortran to match this?

Reply:
Sure! I'll make those changes.